lib/recipe/widgets/ingredient_form.dart

```diff
@@ -242,89 +245,30 @@ class IngredientMeasurements extends StatelessWidget {
                                   measurementIndex: index,
                                   ingredient: state.recipes[recipeIndex]
                                       .ingredients[ingredientIndex],
-                                  measurementUnit: answer!,
+                                  measurementUnit: measurementUnit,
                                   recipe: state.recipes[recipeIndex])),
-                          selectedItem: state
-                              .recipes[recipeIndex]
-                              .ingredients[ingredientIndex]
-                              .measurements[index]
-                              .measurementUnit),
-                      TextFormField(
-                        initialValue: state
-                            .recipes[recipeIndex]
-                            .ingredients[ingredientIndex]
-                            .measurements[index]
-                            .measurementValue,
-                        decoration: InputDecoration(
-                          icon: const Icon(Icons.format_list_numbered_rounded),
-                          labelText: 'Measurement value',
-                          helperText: 'Input measurement value',
-                          errorText: !state
-                                  .recipes[recipeIndex]
-                                  .ingredients[ingredientIndex]
-                                  .measurements[index]
-                                  .isValueValid()
-                              ? 'Enter the measured value in 1 to 4 digits'
-                              : null,
-                        ),
-                        onChanged: (value) {
-                          context.read<RecipeBloc>().add(
-                              IngredientMeasurementValueChanged(
-                                  measurementIndex: index,
-                                  ingredient: state.recipes[recipeIndex]
-                                      .ingredients[ingredientIndex],
-                                  measurementValue: value,
-                                  recipe: state.recipes[recipeIndex]));
-                        },
-                        textInputAction: TextInputAction.next,
-                        keyboardType: TextInputType.number,
-                      ),
-                      const Divider(),
+                          onMeasurementValueChanged: (measurementValue) =>
+                              context.read<RecipeBloc>().add(IngredientMeasurementValueChanged(measurementIndex: index, ingredient: state.recipes[recipeIndex].ingredients[ingredientIndex], measurementValue: measurementValue, recipe: state.recipes[recipeIndex]))),
+                      // TODO: Move the Add Measurement Visibility ListTile
+                      // out of the ListView.builder() so that it is not
+                      // repeated after each measurement (fix bottom overflow)
```

Comment:
With respect to repeating the Add button, its probably down to a design choice. The repeating Add button was intentional on my part so the user doesn't need to scroll to the bottom of the measurements to add a new one

Reply:
Is there a overflow error as well? If so that'll just need to be separately fixed.

Reply:
@adam-sroka @sshazril  was this addressed?